sleeper_wrapper/players.py

```diff
@@ -4,8 +4,11 @@ class Players(BaseApi):
 	def __init__(self):
 		pass
 
-	def get_all_players(self):
-		return self._call("https://api.sleeper.app/v1/players/nfl")
+	def get_all_players(self, sport):
+		if sport == "nfl":	
+			return self._call("https://api.sleeper.app/v1/players/nfl")
+		if sport == "nba":
```

Comment:
IMO this should be `elif sport == "nba"` and then there should be an `else` statement with some sort of logging to let the user know if they provided a `sport` argument that is not supported by the API. I did double check and this call _does_ return NBA players. `lcs` is also supported, so if we're gonna add one, then we should probably add both.